Yosemite/YosemiteTests/Stores/ProductStoreTests.swift

```diff
@@ -1578,6 +1592,16 @@ private extension ProductStoreTests {
         return [defaultAttribute1]
     }
 
+    func sampleDownloadsMutated() -> [Networking.ProductDownload] {
+        let download1 = ProductDownload(downloadID: "1f9c11f99ceba63d4403c03bd5391b11",
+                                        name: "Song #1",
+                                        fileURL: "https://woocommerce.files.wordpress.com/2017/06/woo-single-1.ogg")
```

Comment:
Can you change it to `https://example.com`? 🙇

Reply:
Changed accordingly